atlas/AtlasMl/atlasml/ml/MLPipelines/PipelineWorkflows.py

```
@@ -0,0 +1,292 @@
+            competency = self.weaviate_client.get_embeddings_by_property(CollectionNames.COMPETENCY.value, "cluster_id",
+                                                                         str(labels[index]))
+            if not competency: continue
+            competency_id = competency[0]["properties"]["competency_id"]
+            properties = {
+                "text_id": text_entry["properties"]["text_id"],
+                "text": text_entry["properties"]["text"],
+                "competency_ids": [competency_id]
+            }
+            self.weaviate_client.update_property_by_id(CollectionNames.TEXT.value, text_entry["id"], properties)
+
+        return
```

Ruff (0.11.9)

Useless `return` statement at end of function

Remove useless `return` statement

(PLR1711)